targets/TARGET_WICED/WICED.md

```diff
@@ -0,0 +1,229 @@
+# How to add the WICED SDK
+
+This document summarizes how to download and integrate the WICED SDK into Mbed OS.
+
+**NOTE:** This document is intended for when the WICED SDK is extracted from Mbed OS.
+
+There are several files not provided by the WICED SDK. This is a non-exhaustive
+list of files that are currently provided by Mbed OS:
+- w_generated_mac_address.txt
+- w_generated_security_dct.h
+- w_resources.h
+- mbed_lib.json
+
+## Download the WICED SDK
+
+You can find the full instructions to download the WICED SDK with Advantech's patches
+on Advantech's Wiki:
+http://ess-wiki.advantech.com.tw/view/WISE-1530_SDK
+
+To get the WICED SDK, download WICED-Studio-5.2.0 for your platform:
+https://community.cypress.com/community/wiced-wifi/wiced-wifi-documentation
+
+Once downloaded, move the WICED SDK folder into `targets/TARGET_WICED`.
+
+For the WISE1530, download the patch from Advantech:
+`WM-BN-BM-22_SDK_5.1.x_platform_patch.zip`
+
+Once downloaded, copy the contents into `targets/TARGET_WICED/WICED`.
+
+## Modify the WICED SDK to support Mbed OS's build tools
+
+1. Remove these files because they are unnecessary and large:
+   - `targets/TARGET_WICED/WICED/tools`
+   - `targets/TARGET_WICED/WICED/build`
```

Comment:
There isn't build-directory

Reply:
The build directory is generated if you compile using the WICED studio. I can go ahead and remove this if we want.